.github/kokoro_label_all.yml

```diff
@@ -0,0 +1,21 @@
+# Copyright 2020-present the Material Components for iOS authors. All Rights Reserved.
```

Comment:
Should this file be in here?

Reply:
yea its for copybara-service worker.
https://github.com/material-components/material-components-ios/pull/9503